concordium-sdk/src/main/java/com/concordium/sdk/BlocksAtHeightRequest.java

```diff
@@ -0,0 +1,57 @@
+package com.concordium.sdk;
+
+import lombok.Getter;
+import lombok.ToString;
+
+@Getter
+@ToString
+public class BlocksAtHeightRequest {
+    private final Type type;
+    private final long height;
+
+    private int genesisIndex;
+    private boolean restrictedToGenesisIndex;
+
+    private BlocksAtHeightRequest(Type type, long height) {
+        this.type = type;
+        this.height = height;
+    }
+
+    public BlocksAtHeightRequest(Type relative, long height, int genesisIndex, boolean restrictedToGenesisIndex) {
+        this(relative, height);
+        this.genesisIndex = genesisIndex;
+        this.restrictedToGenesisIndex = restrictedToGenesisIndex;
+    }
+
+    public static BlocksAtHeightRequest newAbsolute(long height) {
```

Comment:
Wouldn't documentation be useful?

Reply:
Code is self explanatory.. 
I will add some :)